.github/workflows/stalebot.yml

```diff
@@ -0,0 +1,26 @@
+name: Close stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 9 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: "It's been a while since we last heard from you. We are marking this issue as stale due to inactivity. Please provide the requested feedback or the issue will be closed after next 7 days."
+        stale-pr-message: "It's been a while since we last heard from you. We are marking this pull request as stale due to inactivity. Please provide the requested feedback or the pull request will be closed after next 7 days."
+        close-issue-message: "There was no activity regarding this issue in the last 14 days. We're closing it for now. Still, feel free to provide us requested feedback so that we can reopen it."
+        close-pr-message: "There was no activity regarding this pull request in the last 14 days. We're closing it for now. Still, feel free to provide us requested feedback so that we can reopen it."
+        stale-issue-label: 'status:stale'
+        stale-pr-label: 'pr:stale'
+        exempt-issue-labels: 'status:confirmed'
+        exempt-pr-labels: 'status:confirmed'
```

Comment:
I wonder if we really need this? We never label PRs, but OTOH it's kind of a workaround if we would like to keep some PRs opened longer... Still I recall we discussed that we shouldn't keep such PRs open and if there is some work going on it will be updated.

I'm for removing it TBH. WDYT?